### PR TITLE
Back end structure

### PR DESCRIPTION
**Pull request by brianthomashammond**
Seeds done, update and destroy done, overall back-end structure done (we are aware of a blocker regarding getting the id of a new search in order to display the search result and would like to use the 1:00 slot to address that)

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -22,16 +21,83 @@ class App extends React.Component {
...
 
+  componentDidMount() {
+    this.getSearches();
+  }
+
+  getSearches = () => {
+    fetch("/ticity_searches")
+      .then((response) => {
+        return response.json();
+      })
```
Comment (zan-rosenthal) [inline app/javascript/components/App.js:36]: Just FYI, you can get rid of curly braces  and `return` keyword and just put response.json() after the fat arrow.

**Review by zan-rosenthal — APPROVED** [review]

File: app/controllers/ticity_searches_controller.rb [hunk 2]
```diff
@@ -1,2 +1,39 @@
 class TicitySearchesController < ApplicationController
+  
+  def index
+    ticity_searches = TicitySearch.all
```
Comment (whitecl) [inline app/controllers/ticity_searches_controller.rb:4]: this action returns all searches for all users. Is scoping this down to just the logged-in user a planned feature?

File: app/controllers/ticity_searches_controller.rb [hunk 3]
```diff
@@ -1,2 +1,39 @@
...
+  def destroy
+    ticity_search = current_user.ticity_searches.find(params[:id])
+    ticity_search.destroy()
+    render json: ticity_search
+  end
+
+  private
+
+  def ticity_search_params
+    params.require(:ticity_search).permit(:city1, :city2, :city3, :housing, :costofliving, :startups, :capital, :travel, :commute, :business, :safety, :healthcare, :education, :environment, :economy, :taxation, :internetaccess, :leisure, :tolerance, :outdoors, :user_id)
```
Comment (whitecl) [inline app/controllers/ticity_searches_controller.rb:36]: Not a blocker: Being able to pass user_id here means that any logged in user could send whatever user_id they want and have a search get created for the passed user_id.

I suggest merging these params with {user_id: current_user.id} in a future PR.

**Review by whitecl — APPROVED** [review]
Great PR, left a couple of non-blocking feedback items

**Pull request merged**